packages/infra/lib/api-stack/api-service.ts

```diff
@@ -133,15 +135,18 @@ export function createAPIService({
   const listenerPort = 80;
   const containerPort = 8080;
   const logGroup = LogGroup.fromLogGroupArn(stack, "ApiLogGroup", props.config.logArn);
+  const desiredTaskCount = isProd(props.config) ? 6 : isSandbox(props.config) ? 2 : 1;
+  const maxTaskCount = isProd(props.config) ? 40 : isSandbox(props.config) ? 10 : 5;
```

Comment:
The idea is to have single CPU tasks, but have more of those. We're currently w/ 2 CPUs running at any given time, now we'll be w/ at least 6.

Reply:
at this point I'd make a function that determines the count - the chained conditionals are getting unwieldy

Reply:
updated